crates/database/src/order_execution.rs

```diff
@@ -123,17 +124,30 @@ mod tests {
             &Default::default(),
             1,
             0,
-            &Default::default(),
+            Asset {
+                amount: BigDecimal::from(3),
+                token: Default::default(),
+            },
             protocol_fees.as_slice(),
         )
         .await
         .unwrap();
 
         // save entry without protocol fees (simulate case when we are still not
```

Comment:
Is this case still relevant?

Reply:
fixed the comment. I think it's still valuable to keep this as part of the test.